### PR TITLE
Clean up unnecessary get_tag_info juggling

### PR DESCRIPTION
**Pull request by jerith**

### CODE REVIEW

**Comment by rudigiesler** [comment]
I'm not sure about the overall view what's happening. Why are we removing the caching? :+1: On the code, looks good to me.

**Comment by jerith** [comment]
We're only getting it through the cache in one place (during routing), so the extra layer isn't buying us anything. The driving force behind the change is cleaning up unnecessary complexity around the message store so we can switch to the new implementation more easily.

**Comment by rudigiesler** [comment]
:+1:

File: go/vumitools/utils.py [hunk 1]
```diff
@@ -188,13 +188,6 @@ def set_tag(self, tag):
         TaggingMiddleware.add_tag_to_msg(self.message, tag)
         self.tag = TaggingMiddleware.map_msg_to_tag(self.message)
 
-    def get_tag_info(self):
-        if self.tag is None:
-            raise ValueError("No tag to look up metadata for.")
```
Comment (JayH5) [inline go/vumitools/utils.py:193]: So now we don't have this `None` check... Is it ok to pass `None` to `get_tag_info` or are we sure we won't do that?

Comment (jerith) [inline go/vumitools/utils.py:193]: We already have a `None` check at the call site. :-)

Comment (JayH5) [inline go/vumitools/utils.py:193]: oh lol. Long day.

**Comment by JayH5** [comment]
:+1: :computer:

**Pull request merged**